CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## securesystemslib v0.17.0
+
+### Added
+* Add `interface.import_publickeys_from_file()` convenience function (#278, #285)
+* Add `gpg.export_pubkeys()` convenience function (#277)
+* Add support to `hash` module for blake2b-256 algorithm (#283)
+
+### Changed
+* Use ecdsa as keytype for ECDSA keys to better distinguish between keytype
+  and scheme(#267)
+* Bump dependencies: cffi (#266), cryptography (#269, #274), colorama (#284)
```

Comment:
Minor nit: (there where two cffi bumps, I guess it's okay too to only use the latter)
```suggestion
* Bump dependencies: cffi (#266, #273), cryptography (#269, #274), colorama (#284)
```